5.py

```diff
@@ -0,0 +1,12 @@
+import numpy as np
+
+"""
+    The program create a 10x10 matrix, in which the elements on the borders will be equal to 1, and inside 0
+"""
+
+x = np.ones((10,10))
+y = np.array([3, 3, 3])
+result = np.empty_like(x)
```

Comment:
This is functional but complicated, search for a numpy command that suits this action without loops.